app/presenters/hyrax/manifest_components.rb

```diff
@@ -0,0 +1,77 @@
+module Hyrax
+  module ManifestComponents
+    def manifest_url
+      manifest_helper.polymorphic_url([:manifest, self])
+    end
+
+    # IIIF rendering linking property for inclusion in the manifest
+    #
+    # @return [Array] array of rendering hashes
+    def sequence_rendering
+      renderings = []
+      if solr_document.rendering_ids.present?
+        solr_document.rendering_ids.each do |file_set_id|
+          renderings << build_rendering(file_set_id)
+        end
+      end
+      renderings.flatten
+    end
+
+    # IIIF metadata for inclusion in the manifest
+    #
+    # @return [Array] array of metadata hashes
+    def manifest_metadata
+      metadata = []
+      metadata_fields.each do |field|
+        metadata << {
+          'label' => field.to_s.humanize.capitalize,
+          'value' => get_metadata_value(field)
+        }
+      end
+      metadata
+    end
+
+    private
+
+      def manifest_helper
+        @manifest_helper ||= ManifestHelper.new(request.base_url)
+      end
+
+      # Build a rendering hash
+      #
+      # @return [Hash] rendering
+      def build_rendering(file_set_id)
+        query_for_rendering(file_set_id).map do |x|
```

Comment:
Would you consider renaming the block variable to be more descriptive? Something like `file_set_document`?

Reply:
I'm not quite understanding?